safe_eth/eth/tests/clients/test_blockscout_client.py

```diff
@@ -10,7 +8,7 @@
 from .mocks import safe_proxy_abi_mock, sourcify_safe_metadata
 
 
-class TestBlockscoutClient(TestCase):
+class TestBlockscoutClient(unittest.TestCase):
```

Comment:
I think is not necessary Django support for this test because there is not any Django model involved.
Extending Django TestCase require a database connection and is not necessary for this test.

Reply:
Agree, are more tests inheriting from Django TestCase?

Reply:
I´ll keep an eye on it.